src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -85,7 +121,28 @@ const AnnotationActivity = ({
                     <div>
                         <ActivityTimestamp date={createdAtTimestamp} />
                     </div>
-                    <ActivityMessage id={id} tagged_message={message} getUserProfileUrl={getUserProfileUrl} />
+                    {isEditing ? (
+                        <CommentForm
+                            isDisabled={isDisabled}
```

Comment:
When would we want to disable this form? It seems like as soon as we submit it, we close the `CommentForm`

Reply:
Yes that's correct - I initially thought I needed the disabling once the user has submitted the form but it looks like that is already being addressed at the parent div level in AnnotationActivity.

Reply:
I can remove this line.